Cargo.toml

```diff
@@ -16,28 +16,29 @@ default = []
 warn_mesh_load = []
 
 [dependencies]
-bytemuck = "1.9"
+bytemuck = "1.13"
 itertools = "0.10"
-bvh = "0.7.1"
+bvh = "0.7.2"
+# TODO: Upgrade to 2.0.2
 bitflags = "1.3"
 serde = "1.0"
-serde_variant = "0.1.1"
+serde_variant = "0.1.2"
 num-traits = "0.2"
 num-derive = "0.3"
 
 [dependencies.bevy]
-version = "0.9.1"
+version = "0.10.1"
 default-features = false
 features = ["bevy_asset", "bevy_render", "bevy_pbr", "bevy_core_pipeline"]
 
 [dev-dependencies]
-bevy = { version = "0.9.1", default-features = true, features = [
+bevy = { version = "0.10.1", default-features = true, features = [
```

Comment:
Is it possible to have `default-features = false` and only include what's needed for `bevy-hikari` to work?